www/js/playerlib.js

```diff
@@ -2194,26 +2144,20 @@ var renderSongs = function(albumPos) {
 	//console.log('LIB.filters.albums=(' + LIB.filters.albums + ')');
 	//console.log('pos=(' + albumPos + ')')
     
-  	// Library panel cover art and metadata
+  // Library panel cover art and metadata
 	if (allAlbums.length == 1 || LIB.filters.albums != '' || typeof(albumPos) !== 'undefined') {
 		$('#lib-coverart-img').html('<a href="#notarget" data-toggle="context" data-target="#context-menu-lib-all">' + 
 			'<img class="lib-coverart" src="' + makeCoverUrl(allSongs[0].file) + '" ' + 'alt="Cover art not found"' + '></a>');
 		$('#lib-albumname').html(allSongs[0].album);
 
-		// r44d for compilation album when using Artist instead of AlbumArtist list ordering
-		if (typeof(albumPos) !== 'undefined') {
-			if (albumPos = UI.libPos[0]) {
```

Comment:
@moodeaudio This is super confusing doing an assignment in an if statement, and I think there might a bug here. Should it be `if (albumPos === UI.libPos[0]) {`? I think my new code follows the same behaviour as the old.